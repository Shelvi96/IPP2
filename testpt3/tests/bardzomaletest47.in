NEW ID

1>;644
406;9;021>;90
:2>0:608
7206892:>2540856959
599719:7>::90;79
:7144>80222;41
5:86>612
;3>:26:
639:19;>4178641447
;>28;592844
:8589>;4;437:;07
;37;672>483018:1
;282:772>95:4198
0>779
714>1712:3302
78951;131>317389
8:78>2:90
05734349>330447
@2:116393631386416;59376::
@76:9:47;662
@98220:08367182060:5;8787;8
@9294;57
@9955540602667836427;23:;578847
@24:79687:2199108:339
@6;0368;;3410609185;;:08014:8
@;95;909:;017112237
@0:5:;624244281;;20803
@2;27
@8:0;22811279
@417066;96;1:071336
@5:280;16503:495874
@8:;:0;5160;:
@94627701;7;44500108
@3:464914932:
@843744;03380781;9567113:60
@962436;23353
