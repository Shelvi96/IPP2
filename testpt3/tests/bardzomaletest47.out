56612019288
0
910401815829
0
9531695078437741
221201
11101100000000
1539
1210020
0
0
1540
5630
0
31000
0
599388206461
0
